diff --git a/components/arduino/CMakeLists.txt b/components/arduino/CMakeLists.txt
index 2e54a060..5f8f64c6 100644
--- a/components/arduino/CMakeLists.txt
+++ b/components/arduino/CMakeLists.txt
@@ -105,7 +105,6 @@ set(LIBRARY_SRCS
   HTTPUpdate
   Insights
   LittleFS
-  Matter
   NetBIOS
   Network
   OpenThread
@@ -508,9 +507,6 @@ endif()
 if(NOT CONFIG_ARDUINO_SELECTIVE_COMPILATION OR CONFIG_ARDUINO_SELECTIVE_ESP_SR)
     maybe_add_component(espressif__esp_sr)
 endif()
-if(NOT CONFIG_ARDUINO_SELECTIVE_COMPILATION OR CONFIG_ARDUINO_SELECTIVE_Matter)
-    maybe_add_component(espressif__esp_matter)
-endif()
 if(NOT CONFIG_ARDUINO_SELECTIVE_COMPILATION OR CONFIG_ARDUINO_SELECTIVE_LittleFS)
     maybe_add_component(joltwallet__littlefs)
 endif()
diff --git a/main/idf_component.yml b/main/idf_component.yml
index 8d6f78e4..53d0cd3e 100644
--- a/main/idf_component.yml
+++ b/main/idf_component.yml
@@ -16,8 +16,3 @@ dependencies:
     version: ">=1.4.2"
     rules:
       - if: "target in [esp32s3]"
-  espressif/esp_matter:
-    version: "1.4.1"
-    require: public
-    rules:
-      - if: "target not in [esp32c2, esp32p4, esp32c61]"
diff --git a/components/arduino/idf_component.yml b/components/arduino/idf_component.yml
index 7c9cf365..f8bf0ce7 100644
--- a/components/arduino/idf_component.yml
+++ b/components/arduino/idf_component.yml
@@ -136,4 +136,1 @@ dependencies:
-  espressif/esp-modbus:
-    version: "^2.1.2"
-    require: public
   joltwallet/littlefs:
